src/Text/Mustache/Types.hs

```diff
@@ -379,3 +382,13 @@ data Template = Template
   , ast      ∷ STree
   , partials ∷ TemplateCache
   } deriving (Show)
+
+instance Lift TemplateCache where
+  lift = const [|HM.empty|]
```

Comment:
Probably doesn't make a difference, but [this](https://github.com/bennofs/th-lift-instances/blob/master/src/Instances/TH/Lift.hs#L128) implementation for `Lift` might avoid any surprises is future.